Coverage/MergeRunner.h

```diff
@@ -59,7 +59,7 @@ class MergeRunner
 
     void merge(const DictCoverage& dictOutput, DictCoverage& dictMerge)
     {
-        auto& itOutput = dictOutput.cbegin();
+        DictCoverage::const_iterator itOutput;
```

Comment:
This is incorrect... the auto& is also incorrect. 

I guess it should be: auto itOutput = dictOutput.cbegin();

Reply:
Fixed